### DISCUSSION TITLE
firefox版插件提示我安装了多个版本的bewlybewly ，但扩展中心只显示了1条，请问如何解决

### DISCUSSION THREAD

**Discussion by rainocean** [BewlyBewly/BewlyBewly#1070] [Q&A]
之前有个时期firefox扩展商店下架了，我就手动安装过一次，后面商店上了又继续从商店安装，近期开始提示我有安装了不同版本，但是我找不到从哪里卸除多余版本，有没有人可以教下我...

**Comment by hakadao** [comment]
ctrl + shift + a 搜尋 BewlyBewly 看看

**Reply by rainocean** [reply]
搜索出来只有1个bewlybewly
![图片](https://github.com/user-attachments/assets/4ba8894e-d8ac-4716-8e8c-1e4da6dd78db)

**Reply by hakadao** [reply]
方便說說firefox是甚麼版本的嗎，我在131.0.3版本中暫時沒有出現此問題
另外可否在在此圖上方點擊齒輪選擇，調試擴展看看是否有還有一個 bewlybewly 在裏面？
如果有的話請嘗試停用或移除
![image](https://github.com/user-attachments/assets/a9d1a339-d7ec-4413-b7e7-4be1db81dbfd)
![image](https://github.com/user-attachments/assets/8255ab23-d0de-4a96-ba98-c35024db5b91)

**Reply by rainocean** [reply]
临时扩展里面没有的

**Comment by hakadao** [comment]
b站dm也有人有一樣的問題，firefox更新擴充功能後直接f5頁面就解決

**Reply by rainocean** [reply]
好的收到 谢谢您
